### PR TITLE
Share buttons added in news section

### LINKED ISSUE #525: Add share buttons to news items
So that people can easily share these pages via Twitter/FB/&c. 

Here are some resources that could help for adding buttons to each post: 
https://superdevresources.com/share-buttons-jekyll/
https://jekyllcodex.org/without-plugin/share-buttons/

### PR DESCRIPTION
**Pull request by richa208**
Fixes #525 

hot11

@ramyaragupathy @smit1678 Kindly review this.

### CODE REVIEW

**Comment by bismitaguha** [comment]
Hey @richa208 considering the uniformity of the website I suggest the buttons of share to be of the same hex colour as being used throughout the site.
Also share options should be in the left section in the div which is fixed, for better UX.

**Review by bismitaguha — CHANGES_REQUESTED** [review]
Share options should be fixed on scroll for better user-experience. Also their colours should be uniform with the website colours.

**Comment by richa208** [comment]
Hi @codefab99 
I included those icons for a change. IMO UX can vary according to every person as they feel appropriate.
So I think @ramyaragupathy @dakotabenjamin @smit1678 can best tell what design would be appropriate here.

**Comment by bismitaguha** [comment]
Sharing with ease should be there. I was **suggesting** fixed div as an option. Design can have various approaches and gets better with discussions. @richa208 
I also feel since the header footer has icons in a uniform colour, using those icons will maintain the site uniformity

File: _layouts/news-item.html [hunk 1]
```diff
@@ -110,3 +110,33 @@ <h5>Associated Projects</h5>
...
+    <a  href="https://www.facebook.com/sharer.php?u={{ site.url }}{{ page.url }}&title={{ page.title }}"
+        onclick="window.open(this.href, 'pop-up', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"
+            title="Share on Facebook" >
+      <img id="share-buttons" src="/images/facebook.png" alt="Facebook" ></img>
+    </a>
+
+    <a  href="https://twitter.com/intent/tweet?text={{ page.title }}&url={{ site.url }}{{ page.url }}&via={{ site.twitter_username }}&related={{ site.twitter_username }}"
+            onclick="window.open(this.href, 'pop-up', 'left=20,top=20,width=500,height=500,toolbar=1,resizable=0'); return false;"
+            title="Share on Twitter" >
+      <img id="share-buttons" src="/images/Twitter.png" alt="Twitter" ></img>
```
Comment (bismitaguha) [inline _layouts/news-item.html:127]: Using fa icons is preferable rather than png.

File: _sass/base.scss [hunk 2]
```diff
@@ -5079,5 +5079,25 @@ sup {
...
+  padding-bottom : 15px;
+}
 
+#share-bar h4 {
+  margin-bottom: 10px;
+  font-weight: 500;
+}
 
+#share-bar a{
+  color:#fff !important;
```
Comment (bismitaguha) [inline _sass/base.scss:5094]: Using !important in CSS is not a good practice.

**Review by bismitaguha — CHANGES_REQUESTED** [review]

**Review by ramyaragupathy — COMMENTED** [review]
Thanks for working on this @richa208! Adding my thoughts here:
- We already have most of the social media icons in our [image resources](https://github.com/hotosm/hotosm-website/tree/gh-pages/images). Please try using the same for the share feature
-  A few ideas on share feature placement within the page:
Should we have feature as a sticky menu on the left sidebar, similar to `Associated Projects` in the gif attached?
sticky


Your current icon placement is at the end of the page right above the footer and on a mobile view this appears  below the side bar content
image.
So just adding these icons to the side bar as a sticky menu wouldn't make a difference for the mobile view, but it would add to the user engagement on a desktop.

- However placement of icons is also dependant on how we want the share to operate on our page. Do we go with a modal/ tab /expandable div just below the icons?
In our case share feature transition is temporary and the focus of the user has to return to our news page/website after transmitting the information. So a new tab option is ruled out. We're left with either a modal or a div. A modal works good in desktop. However certain mobile browsers might not support modal well. I see the modal opening as a new tab while simulating the experience in a desktop browser. 
Based on this, I am leaning towards offering the share content feature as an expandable div within the same page. In that case your current placement of icons should work good so that we have enough room to accommodate a div with the share content. 

@richa208 - I would like to hear your thoughts on how we should offer the share feature considering end users on different platforms(mobile & desktop). Our key goal is increasing user engagement and retain user stay within website.

**Comment by ramyaragupathy** [comment]
Stale - let's revisit based on user interviews

**Pull request closed without merging**